### ISSUE TITLE
add run scripts to package.json

### ISSUE THREAD

**Issue by hyzhak** [UtsavPreet/minorProject-docBot#2]
- [x] for start firebase server
- [x] deploy to server
- [ ] test #3 
- [x] lint #4

**Comment by UtsavPreet** [comment]
Thank You so much for raising this issue, specially the test one; I have never used any test frameworks, but it was a good time learning about them, will try and write my first test soon.

**Comment by hyzhak** [comment]
@UtsavPreet yes, they are very useful especially for open projects where anyone can push commits and we should keep its consistency. And btw I'm glad to see that you are trying to learn and don't give up. It is a very useful skill for a software engineer.

**Comment by UtsavPreet** [comment]
@hyzhak Yeah, indeed now I understand the whole idea behind having the test cases, and this whole exercise helped me a lot. So a big thanks to you for giving me such challenging tasks. And thanks for the compliment, really appreciate that 😄.
BTW, with the final proposal date for GSOC'18 round the corner, how should I proceed with that ?

**Comment by hyzhak** [comment]
@UtsavPreet I don't see your proposal in GSoC dashboard, have you sent it? If not please send to I could discuss your proposition with colleagues. Big thanks!

**Comment by UtsavPreet** [comment]
@hyzhak yes I haven't send one, because I wanted to know if I can contribute to the project, through my skill set, I'll create one and add today, Any specific proposal template ?
